Update Readme for iOS platform specific info (#2473)
rust-windowing/winit: README.md

If you encounter problems, you should try doing your initialization inside
`Event::NewEvents(StartCause::Init)`.

#### iOS

> Similiar like MacOS, iOS's main `UIApplicationMain` did some init work that required
> by all UI related code. see issue [#1705], you should create Window inside event loop.

Reviewer: Nit:

```suggestion
Similar to macOS, iOS's main `UIApplicationMain` does some init work that's required
by all UI related code, see issue [#1705]. You should consider creating your windows
inside `Event::NewEvents(StartCause::Init)`.
```
